src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback } from "react";
-import { useKeyEvent } from "hooks";
+import { useKeyEvent } from "../../hooks";
```

Comment:
@hadasfa lets fix this file - and make sure that we import it directly from the file and not the general expose index

Reply:
added to the backlog as first step. I will try to take this next iteration